cmd/eksctl-anywhere/cmd/installpackages.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/config"
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+type installPackageOptions struct {
+	source      curatedpackages.BundleSource
+	kubeVersion string
+	packageName string
+	registry    string
+}
+
+var ipo = &installPackageOptions{}
+
+func init() {
+	installCmd.AddCommand(installPackageCommand)
+	installPackageCommand.Flags().Var(&ipo.source, "source", "Location to find curated packages: (cluster, registry)")
+	if err := installPackageCommand.MarkFlagRequired("source"); err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+	installPackageCommand.Flags().StringVar(&ipo.kubeVersion, "kubeversion", "", "Kubernetes Version of the cluster to be used. Format <major>.<minor>")
+	installPackageCommand.Flags().StringVarP(&ipo.packageName, "packagename", "p", "", "Custom name of the curated package to install")
```

Comment:
We can camelcase packagename and kubeversion right?

Reply:
This is client's flag. I believe other commands use the same format as above. For example filename instead of fileName for the flag.

Reply:
I think packageName is better because it disambiguates with the name of the installation.